com.unity.multiplayer.mlapi/Runtime/Core/RPCQueue/QueueHistoryFrame.cs

```diff
@@ -0,0 +1,222 @@
+using System.Collections.Generic;
+using MLAPI.Serialization.Pooled;
+
+namespace MLAPI
+{
+    /// <summary>
+    /// QueueHistoryFrame
+    /// Used by the RpcQueueContainer to hold queued RPCs
+    /// All queued Rpcs end up in a PooledBitStream within a QueueHistoryFrame instance.
+    /// </summary>
+    public class QueueHistoryFrame
+    {
+        public enum QueueFrameType
+        {
+            Inbound,
+            Outbound,
+        }
+
+        public uint            TotalSize;
+        public List<uint>      queueItemOffsets;
+
+        public PooledBitStream queueStream;
+        public PooledBitWriter queueWriter;
+        public PooledBitReader queueReader;
```

Comment:
minor: first letters might be capitalized for public fields

Reply:
This is part of the standard.